azure-pipelines.yml

```diff
@@ -159,6 +159,7 @@ steps:
     echo "BUILD_REASON:"$BUILD_REASON
     echo "cmd: "$(cmd)
     eval "$(cmd)"
+    ./.ci/travis.sh git-diff
```

Comment:
Move all to validation.sh as it was in previous commit.
We should not use tavis related commands in other CIs